guide/payments/request.md

```diff
@@ -116,52 +108,126 @@ A piece of meta data unique to invoices is an expiration time. This is how long
    image = "/assets/images/guide/payments/request/single-use-payment-request.png"
    retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
    layout = "shadow"
-   caption = "A newly generated single-use Lightning payment request."
+   caption = "A newly generated single-use payment request."
    alt-text = "Screen showing single-use payment request."
    width = 250
    height = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/payments/request/single-use-payment-request-settings.png"
-   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings@2x.png"
+   image = "/assets/images/guide/payments/request/long-expiry.png"
+   retina = "/assets/images/guide/payments/request/long-expiry@2x.png"
    layout = "shadow"
-   caption = "Let users configure the settings of single-use Lightning payment requests."
-   alt-text = "Screen showing single-use payment request settings."
+   caption = "Zero value payment requests should be easily generated."
+   alt-text = "Screen showing invoice with long expiry."
    width = 250
    height = 541
 %}
 
+</div>
+
+### Invoice expirations
+
+Single-use lightning invoices aren't permanent; they expire over time. By default this expiry is set to 60 minutes. However, it is possible to adjust this expiration time which unlocks unique use cases and improves usability in some situations.
+
+So what is the ideal invoice expiry time to set for your application? Well it depends, bitcoin UX designer Stephen DeLorme goes into depth on this issue [here](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/). If your users are denominating their payments in bitcoin, long expiry times such as 24 hours should be used. It is only when denominating in fiat that expiry times become a prominent usability issue.
+
+{% include /tip/open.html label="Why do invoices expire?" icon="info" color="blue" %}
+
+If invoices had no expiry, recipients would likely to run into memory / storage issues as the number of locally stored preimages grows with each payment attempt. Rene Pickhardt explains this further [here](https://bitcoin.stackexchange.com/questions/85981/why-do-lightning-invoices-expire/85999#85999).
+
+{% include /tip/close.html %}
+
+#### Short expiry times
+
+Short expiry times, an example being 60 seconds, are ideal for in-person payments that are done quickly. These can stress users out as they attempt to make the payment on time, or not make the payment at all due to the short payment window resulting in another request needing to be generated.
```

Comment:
60 seconds is such a weird default. Does the price of bitcoin really change that much in 60 seconds? Why not 5 or 10 minutes instead so people don't have to stress out.

Reply:
They shouldn't be exposed to the expiration time at such short time frames. Rather it should refresh every 60 seconds behind the scenes without either payment party having to worry about it. I'll articulate this better.